CONTRIBUTING.md

```diff
@@ -1,17 +1,64 @@
 # Contribution guidelines
 
-Brave welcomes contributions via [pull requests](https://github.com/brave/browser-laptop/pulls). Below are a few guidelines we ask of contributors:
-
-## Before you make changes
-
+Table of contents
+- [How can you contribute?](#how-can-you-contribute)
+  - [Help triage issues](#help-triage-issues)
+  - [Updating documentation](#updating-documentation)
+  - [Help with translations](#help-with-translations)
+  - [Work on the code](#work-on-the-code)
+- [Getting started](#getting-started)
+  - [Making changes](#making-changes)
+  - [Pull requests](#pull-requests)
+    - [Considerations before submitting a pull request](#considerations-before-submitting-a-pull-request)
+    - [Each pull request should include](#each-pull-request-should-include)
+    - [Employees should](#employees-should)
+  - [Closing issues](#closing-issues)
+  - [Triage help](#triage-help)
+
+## How can you contribute?
+Brave welcomes contributions of all kinds! You can make a huge impact without writing a single line of code
+
+### Help triage issues
+One of the easiest ways to help is to [look through our issues tab](https://github.com/brave/browser-laptop/issues)
+* Does the issue still happen? Sometimes we fix the problem and don't always close the issue
+* Are there clear steps to reproduce the issue? If not, let's find and document some
+* Is the issue a duplicate? If so, share the issue that is being duplicated in the conversation
+* See our [Triage of issues page](https://github.com/brave/browser-laptop/wiki/(WIP)-Triage-of-issues) for more info about this process
+
+### Updating documentation
+Documentation is extremely important. There are lots of areas we can improve:
+* Having more clear or up-to-date instructions in our [README.md](https://github.com/brave/browser-laptop/blob/master/README.md).
+* Capturing/updating helpful information [in our wiki](https://github.com/brave/browser-laptop/wiki). For example:
+  * [Troubleshooting steps](https://github.com/brave/browser-laptop/wiki/Troubleshooting)
+  * [Frequently Asked Questions](https://github.com/brave/browser-laptop/wiki/End-User-FAQ)
```

Comment:
The link could be updated to `https://community.brave.com/c/common-issues`. Currently the FAQ is a wiki and can be edited anyone, while on the community the topics can be edited only by employees :-)

Reply:
good catch- I removed that from here since that section is basically about updating the wiki (which the FAQ is no longer a part of)